VERSIONS.md

```diff
@@ -20,5 +20,5 @@ Archetype Version   | AEM Version
 20, 21, 22          | 6.5, 6.4, 6.3 + SP3
 23                  | 6.5, 6.4, 6.3 + SP3, AEM as a Cloud Service
 24, 25, 26, 27      | 6.5.5, 6.4.8.1, AEM as a Cloud Service
-28, 30, 31, 32, 33, 34, 35, 36, 37, 39, 40, 41, 42, 43, 45, 46  | 6.5.7, AEM as a Cloud Service
-47                  | 6.5.17, AEM as a Cloud Service
+28, 30, 31, 32, 33, 34, 35, 36, 37, 39, 40, 41, 42, 43, 44  | 6.5.7, AEM as a Cloud Service
```

Comment:
Why have we added 44? cc: @pankaj-parashar @barshatr

Reply:
The entry for 44 was removed mistakenly in one of earlier Version updates, so corrected it here.